services/certificate-api/package.json

```diff
@@ -3,11 +3,13 @@
   "version": "1.0.0",
   "main": "main.js",
   "license": "GPL",
+  "type": "module",
   "scripts": {
     "start": "node main.js",
     "test": "jest"
   },
   "dependencies": {
+    "@cord.network/sdk": "^0.7.5-0",
```

Comment:
latest version where '@cord.network/vc-export' package is also working is "^0.7.9-1"